### PR TITLE
Rework to support renderer for multipoint geojson:

### PR DESCRIPTION
**Pull request by rsjones**
* Single location now for default symbology
* Reworked isTable utility function into new helper pattern
* Removed Utils.getGeomType in favor of newer helper pattern function
* Ensure tamplates.renderLayer data has a features array so computeExtent doesn't throw error
* Add unit tests for is-geojson-table and templates

### CODE REVIEW

File: lib/helpers/is-geojson-table.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+const getGeometryTypeFromGeojson = require('./get-geometry-type-from-geojson')
+
+module.exports = function isTable (data = {}) {
+// geometry indicates this in not a table
+  const geometryType = getGeometryTypeFromGeojson(data)
+  if (geometryType) return false
+
+  // Check for a valid fullExtent. If unset, assume this is a Table
+  const fullExtent = data.fullExtent || (data.metadata && data.metadata.fullExtent)
+  if (_.isUndefined(fullExtent) || _.isUndefined(fullExtent.xmin) || _.isUndefined(fullExtent.ymin) || fullExtent.xmin === Infinity) return true
```
Comment (rgwozdz) [inline lib/helpers/is-geojson-table.js:11]: Can you move all of this to a helper called `hasValidFullExtent`?  Then you could just do:

`return hasValidFullExtent(args)`

File: lib/templates.js [hunk 2]
```diff
@@ -31,12 +34,16 @@ function renderLayer (data = {}, { params = {}, query = {} } = {}) {
   const json = _.cloneDeep(templates.layer)
   const metadata = data.metadata || {}
   const capabilities = data.capabilities || {}
+  // force an empty array of features if missing
+  if (!data.features) {
+    data.features = []
+  }
```
Comment (rgwozdz) [inline lib/templates.js:40]: I'm worried about side-effects of mutating the input `data` argument (not convinced we capture everything with our tests). Is this really needed? It looks like `getGeometryTypeFromGeojson` already handles the case where `features` is not defined.

Comment (rsjones) [reply lib/templates.js:40]: This needs to be done or the extent issue as to why it throws an exception needs to be looked into. As it was existing code, it seemed like a TODO vs something to fix for this.

File: lib/helpers/is-geojson-table.js (hunk 1 above)
Comment (rsjones) [reply lib/helpers/is-geojson-table.js:11]: This was existing code just moved from the utils class.

Comment (rgwozdz) [reply lib/helpers/is-geojson-table.js:11]: It's a pretty small ask.

File: lib/templates.js (hunk 2 above)
Comment (rgwozdz) [reply lib/templates.js:40]: Can you link me to where

```js
if (!data.features) {
    data.features = []
}
```
is found in the existing code?

**Comment by rgwozdz** [comment]
CI is failing due to a linting issue.

**Pull request merged**